docs/src/main/diataxis/doc-concept.adoc

```diff
@@ -0,0 +1,50 @@
+////
```

Comment:
Wondering if the suffix should be `concepts`? Chances are you will introduce more than one concept per doc?

Reply:
yes. that can be done.